doc/reference_accessor.xml

```diff
@@ -1773,11 +2155,79 @@ SELECT * FROM ST_IsValidDetail('LINESTRING(220227 150406,2220227 150407,222020 1
 	  </refsection>
 	</refentry>
 
+	<refentry id="ST_MemSize">
+	  <refnamediv>
+		<refname>ST_MemSize</refname>
+
+		<refpurpose>Returns the amount of memory space a geometry takes.</refpurpose>
```

Comment:
`bytes, serialized, uncompressed.` it's a size of GSERIZLIZED which gets compressed on disk, and decompressed even more in memory when becoming LWGEOM or GEOS geometry. Many postgres sizes are in 8kb pages which means bytes should be mentioned.

Reply:
Not following.  Can you provide some text for this?